src/app/components/error/index.html

```diff
@@ -1,13 +1,196 @@
+            <sky-docs-visual-example-image>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-option-construction.png" />
+                </div>
+            </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+            <sky-docs-visual-example-text>
+              <p>Use the not found error type when there is an error loading a page due to a 404 error.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-option-notfound.png" />
+                </div>
+            </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <sky-docs-visual-example>
+            <sky-docs-visual-example-text>
+              <p>Use the security error type when a user doesn’t have access to view content.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-option-security.png" />
+                </div>
+            </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+        <h3>Image and action</h3>
+        
+        <sky-docs-visual-example>
+            <sky-docs-visual-example-text>
+                <p>Include the optional image and action when the error component is the only content on the page. Tiles have limited space and may not be important to users' immediate tasks, so exclude the images and actions and only display error messages.</p>
+            </sky-docs-visual-example-text>
+            <sky-docs-visual-example-image>
+                <div>
+                  <img src="~/assets/img/guidelines/error/error-usage-6.png" />
+                    <div class="caption caption-success sky-text-success">Keep the error message more subtle when it affects only a portion of the content on a page.</div>
+                </div>
+            </sky-docs-visual-example-image>
+        </sky-docs-visual-example>
+        
+    </sky-demo-page-content>
+        
+    <sky-demo-page-content sectionHeading="Content">
+        
+        <h3>Style and tone</h3>
+        
+            <div class="sky-text-block">
+                <p>Make error and warning messages brief and as accurate as possible to help the user address the problem, and use a conversational approach. When possible, offer a solution along with an explanation of why the error or warning is occurring. Use the error component to provide a consistent user experience with error messages.</p>
+                <p>Errors can frustrate users and sometimes indicate serious problems, so be thoughtful and helpful with the wording. Limit the use of "please" and "sorry," be careful about how you use "can't" and "don't" (avoid "you can't" and "you don't"), and avoid humor.</p>
```

Comment:
I'd remove the conditionals here: "Errors frustrate users and indicate serious problems, so error messages must be thoughtful and helpful."

And in the second sentence, I'd tweak the order so the "avoid humor" doesn't get lost at the end because that reads awkwardly. And the middle item is a little vague, so I'd edit it to be more direct: "Avoid humor, limit the use of 'please' and 'sorry,' and avoid negative constructions such as 'you can't' and 'you don't.' "